UefiPayloadPkg/SPI/pci_ops.h

```diff
@@ -0,0 +1,64 @@
+#ifndef PCI_OPS_H
```

Comment:
Use PciLib from EDK2 and delete this file

Reply:
@IgorBagnucki

Reply:
Done